app/Models/Ldap.php

```diff
@@ -119,6 +120,25 @@ public static function findAndBindUserLdap($username, $password)
 
         \Log::debug('Filter query: '.$filterQuery);
 
+	// userDn should be independent from baseDn (maybe you want to search in >=2 subtrees)
+	// -> better ask LDAP for user dn, that's why it is for
+	if ($settings->is_ad != '1') {
+		$userresults = ldap_search($connection, $baseDn, $filterQuery);
+		$userentries = ldap_get_entries($connection, $userresults);
+		// Can be empty if user does not exist
+		if ( $userentries["count"] > 0 ) {
```

Comment:
I'd prefer this actually be ` == 1` - so if you managed to find two identical-ish looking user, it would refuse to log you in.

Reply:
I changed the condition to ( $userentries["count"] == 1 ) s. also https://github.com/snipe/snipe-it/pull/11715